zenaton/engine.py

```diff
@@ -23,7 +23,8 @@ def __init__(self):
         @return [Array<String>, nil] the results if executed locally, or nil
     """
     def execute(self, jobs):
-        map(Engine._check_argument, jobs)
```

Comment:
In Python 3, `map` is lazy and does not check anything. It's more pythonic to write the for loop anyway.